daprdocs/content/en/reference/components-reference/supported-name-resolution/_index.md

```diff
@@ -7,6 +7,8 @@ description: The supported name resolution providers to enable Dapr service invo
 no_list: true
 ---
 
+Name resolution is configured via the [Dapr Configuration]({{< ref configuration-overview.md >}}). 
```

Comment:
I would put this sentence after the one below - "The following components provide name resolution for the service invocation building block."
```suggestion
Name resolution components are configured via the [configuration]({{< ref configuration-overview.md >}}). 
```